packages/ui/react-ui-stack/src/components/Section.tsx

```diff
@@ -77,18 +77,21 @@ export const Section = forwardRef<HTMLLIElement, SectionProps>(
               {children}
             </div>
 
-            <Button
-              variant='ghost'
-              classNames={[
-                'self-stretch justify-start rounded-is-none',
+            <Card.Menu
+              position={undefined}
+              classNames={mx(
+                fineButtonDimensions,
+                'self-stretch justify-center rounded-is flex items-center bs-auto is-auto',
                 hoverableFocusedControls,
```

Comment:
For some reason the hover behavior changed such that the menu shows all of the time.

Reply:
fixed it